unit-tests-java8/src/test/java/org/eclipse/collections/test/FixedSizeIterableTestCase.java

```diff
@@ -14,7 +14,7 @@
 
 import org.junit.jupiter.api.Test;
 
-import static org.eclipse.collections.impl.test.Verify.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
Does the code compile if you don't swap this import?

Reply:
Revertable now - updating.